packages/blade/src/tokens/global/breakpoints.ts

```diff
@@ -1,25 +1,69 @@
 export type Breakpoints = Readonly<{
-  /** max width: 320px  */
+  /**
+   * `base` is used for base styling. It starts from 0px till the next existing token
+   *
+   * Think of this as styles without any media query.
+   *
+   * ### Example
+   *
+   * This code will set margin as `spacing.2` on "m" size screens and above. And as `spacing.1` on less than "m" size screens
+   * ```jsx
+   * <Box margin={{ base: 'spacing.1', m: 'spacing.2' }} />
+   * ```
+   *
+   * This roughly translates into -
+   *
+   * ```js
+   * styled.div(() => ({
+   *   margin: 'spacing.1',
+   *   [`@media screen and (min-width: 768px)`]: {
+   *     Box: {
+   *       margin: 'spacing.2'
+   *     }
+   *   }
+   * }))
+   * ```
```

Comment:
Perhaps you can add effective CSS instead of `styled` API

Reply:
wasn't sure how to represent spacing.2 and Box but I guess I can add pseudo css code

Reply:
Yup, maybe use a css variable to represent it